OpenUtau.Core/Format/Formats.cs

```diff
@@ -16,6 +16,7 @@ public static class Formats {
         const string midiMatch = "MThd";
         const string ufdataMatch = "\"formatVersion\":";
         const string musicxmlMatch = "score-partwise";
+        const string svpMatch = "\"version\":";
```

Comment:
This is too generic to be used to match a format. Try find something more specific.